src/utils/env.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path')
+const fs = require('fs')
+const { promisify } = require('util')
+
+const fileStat = promisify(fs.stat)
+
+async function getEnvFile(projectDir) {
+  const envDevelopmentFile = path.resolve(projectDir, '.env.development')
+  const envFile = path.resolve(projectDir, '.env')
+
+  try {
+    if ((await fileStat(envDevelopmentFile)).isFile()) return envDevelopmentFile
```

Comment:
That could be used too: https://github.com/sindresorhus/locate-path#usage

```js
return locatePath([envDevelopmentFile, envFile])
```

Reply:
👍🏽